doc/content/docs/imjoy.md

```diff
@@ -47,12 +47,18 @@ function encodeScijsArray(array){
 
 The `image` key is optional; one can also call `setImage()` later.
 
+Context `config`:
```

Comment:
Maybe list all the available config options?

Reply:
@oeway great idea! :bulb: Yes, added in 63d240e. Since we will incrementally add many fields, I created a new page.

Reply:
Great!